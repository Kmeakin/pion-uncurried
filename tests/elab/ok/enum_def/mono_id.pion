enum Id {
    MkId(_: Bool),
}

let get = fn(x) => match x {
    MkId(z) => z,
};