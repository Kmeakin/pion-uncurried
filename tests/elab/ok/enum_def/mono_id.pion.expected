enum Id: Type {
    MkId(_: Bool): Id,
}

let get: fn(x: Id) -> Bool = fn(x: Id) => match x {
    MkId(z) => z,
};
